src/amp-story-ad-preview.js

```diff
@@ -129,4 +162,38 @@ export default class AmpStoryAdPreview {
     // b) reject when invalid
     this.writeToIframe_(await this.adIframe_, patch(dirty));
   }
+
+  async setMetaCtaLabel_(dirty) {
+    const {ctaType, ctaUrl} = this.extractMetaCta_(dirty);
```

Comment:
Nit: Maybe just name these `{type, url}` since you're only calling them from a `cta` context? It's good to reduce prefixes when appropriate to reduce reading length/cognitive load.